numpy/core/src/umath/_struct_ufunc_tests.c.src

```diff
@@ -53,6 +47,58 @@ static void add_uint64_triplet(char **args, npy_intp *dimensions,
     }
 }
 
+static PyObject*
+register_fail(PyObject* NPY_UNUSED(self), PyObject* NPY_UNUSED(args))
+{
+    PyObject *add_triplet;
+    PyObject *dtype_dict;
+    PyArray_Descr *dtype;
+    PyArray_Descr *dtypes[3];
+    int retval;
+
+    add_triplet = PyUFunc_FromFuncAndData(NULL, NULL, NULL, 0, 2, 1,
+                                    PyUFunc_None, "add_triplet",
+                                    "add_triplet_docstring", 0);
+
+    dtype_dict = Py_BuildValue("[(s, s), (s, s), (s, s)]",
+        "f0", "u8", "f1", "u8", "f2", "u8");
```

Comment:
NIT:  at least 2 tabs is typical I think in numpy code base.

Reply:
aligning